packages/koenig-lexical/src/components/KoenigCardWrapper.jsx

```diff
@@ -59,8 +54,9 @@ const KoenigCardWrapper = ({nodeKey, width, wrapperStyle, IndicatorIcon, childre
                         const cardNode = $getNodeByKey(nodeKey);
                         const clickedDifferentEditor = !cardNode;
                         const clickedToolbar = event.target.closest('[data-kg-allow-clickthrough="false"]');
+                        const clickedSettingsPanel = event.target.closest('[data-testid="settings-panel"]');
```

Comment:
Ideally we would not use testid's in any non-test code, there's no guarantee they'll always be present - e.g. the test id's in our Admin app get stripped out in production builds to save some bytes in the build output - and it creates some potentially invisible coupling as there's usually no expectation that test-specific code would be in use elsewhere.

Better to add our own attributes for selecting anything in app code, similar to the existing `data-kg-slash-menu`,  `data-kg-card-menu-item`, `data-kg-allow-clickthrough` etc